### PR TITLE
Add styling for contact section

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
Add three new sections: Contact info section, a Navigation bar, and a Footer.
I accidentally merged a pull request which had a code review with a required change that says: "add a date below the navigation bar and you should use Luxon to get the current date and time" which my partner and I have fixed.

### CODE REVIEW

File: contact.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable linebreak-style */
+const contact = document.querySelector('.contact');
```
Comment (Emmyn5600) [inline contact.js]: - [x] You have disabled too many lines of code while you were trying to fix eslint errors, kindly try to fix eslint errors without disabling many lines of code. 

- check the `index.js and contact.js file`

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Ugboaja-Uchechi  and @VICTOR-CHUKWU 

Great work on implementing the changes requested by the previous reviewer, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- The live link demo is working properly :heavy_check_mark:

Changes required :writing_hand: 

  -  You have to work on Readme file to be more professional for example, adding Author's account, Getting started  as well as live link demo.  
  - Please check the comment under the review.
  
  Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Ugboaja-Uchechi** [comment]
Fix code reviews as best as my partner and I could.

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @VICTOR-CHUKWU and @Ugboaja-Uchechi 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Live link demo is working properly:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewers.

 #### Optional suggestions
 
  - You have to work on Readme file to be more professional for example, adding Author's account, Getting started as well as live link demo.
  
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**